components/Header/Subcomponents/NavigationMenu/NavigationMenu.unit.test.jsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import NavigationMenu from '.';
+
+test('should match the snapshot', () => {
+  const { container } = render(<NavigationMenu />);
+  expect(container).toMatchSnapshot();
+});
+
+it('should render the links and sandwitch button', () => {
+  render(<NavigationMenu />);
+
+  const cathoComponentsLinks = screen.getAllByText(/Catho Components/i);
+  const cathoComponentsVisibleMenuLink = cathoComponentsLinks[0];
+  const cathoComponentsMobileMenuLink = cathoComponentsLinks[1];
+
+  const quantumLinks = screen.getAllByText('Quantum');
+  const quantumVisibleMenuLink = quantumLinks[0];
+  const quantumMobileMenuLink = quantumLinks[1];
+
+  const sandwitchBtn = screen.getByTestId('sandwichButton');
+  const simulatesRendersLink = screen.getByText('Simulates renders');
+  const repositoriesLink = screen.getByText('Repositories');
```

Comment:
These links are inside the menu, right? If so, put the suffix menu similar to the previous links. Standardization makes code easier to read
```suggestion
  const simulatesRendersMenuLink = screen.getByText('Simulates renders');
  const repositoriesMenuLink = screen.getByText('Repositories');
```